checkov/gitlab_ci/runner.py

```diff
@@ -1,22 +1,30 @@
 from __future__ import annotations
 
+import os
 from typing import TYPE_CHECKING, Any
 
-from checkov.common.images.image_referencer import ImageReferencer, Image
+from checkov.common.output.report import Report
+
+from checkov.runner_filter import RunnerFilter
+
+from checkov.common.images.image_referencer import Image, ImageReferencerMixin
 from checkov.common.bridgecrew.check_type import CheckType
 from checkov.gitlab_ci.checks.registry import registry
+from checkov.gitlab_ci.image_referencer.manager import GitlabCiImageReferencerManager
 from checkov.yaml_doc.runner import Runner as YamlRunner
 
 if TYPE_CHECKING:
     from checkov.common.checks.base_check_registry import BaseCheckRegistry
     from collections.abc import Iterable
+    from networkx import DiGraph
 
 
-class Runner(YamlRunner, ImageReferencer):
+class Runner(ImageReferencerMixin, YamlRunner):
     check_type = CheckType.GITLAB_CI  # noqa: CCE003  # a static attribute
 
     def __init__(self) -> None:
         super().__init__()
+        self.file_path = ''
```

Comment:
is it a left over? I don't see any usage, except setting it in method `_parse_file`.

Reply:
yep, good catch 😄